internal/sessiontest/logs_test.go

```diff
@@ -35,27 +36,34 @@ func TestLogging(t *testing.T) {
 	require.NoError(t, err)
 	require.NotEmpty(t, disclosed)
 
+	time.Sleep(2 * time.Second)
```

Comment:
Do we need this explicit sleep here? Aren't there better options?

Reply:
The sleep is needed to make sure that the next session always falls in the next unix time epoch. Otherwise the test on the before parameter of `LoadLogs` might fail. It would be possible to do it in another way, but then we are testing it in a slightly other way than it is used in production. In production the LoadLogs also "assumes" a unix time in seconds is unique for a session. I added a comment with an explanation in a new commit.